import_3dm/converters/render_mesh.py

```diff
@@ -0,0 +1,55 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
The license of import_3dm is MIT, see the [LICENSE](https://github.com/jesterKing/import_3dm/blob/master/LICENSE) file. This comment block needs to be replaced with the correct text.